### PR TITLE
Updates screenshoots for 2.18

### PR DESCRIPTION
**Pull request by SrNetoChan**
This pull request aims to updates the users manual screenshots to 2.18 version.

Many screenshots are outdated because of changes or new features. I will swipe the all document and compare the images with the interface. Hope I can do them all...

**Please wait until it's finished to review or merge**

### CODE REVIEW

**Comment by DelazJ** [comment]
Great plan!!!

File: qgis-projects/user_manual/introduction_qgis_gui_start_up.qgs [hunk 1]
```diff
@@ -50,11 +53,14 @@
...
+        <ymin>1367760</ymin>
+        <xmax>4897040</xmax>
+        <ymax>7809680</ymax>
+      </extent>
       <id>landcover20130516233229406</id>
-      <datasource>../../../qgis_sample_data/raster/landcover.img</datasource>
-      <shortname></shortname>
-      <title></title>
-      <abstract></abstract>
+      <datasource>../../../../Geodata/qgis_sample_data/raster/landcover.img</datasource>
```
Comment (DelazJ) [inline qgis-projects/user_manual/introduction_qgis_gui_start_up.qgs]: @SrNetoChan I know that we should not review that until you allow us (sorry!), but just wonder if changing the datasource wouldn't break projects in other contributors' side (--> Handle bad layers dialog)? I mean qgis_sample_data is supposed to be in the same folder than QGIS-Documentation according to https://github.com/qgis/QGIS-Documentation/blob/release_2.18/qgis-projects/user_manual/readme.txt

Comment (SrNetoChan) [inline qgis-projects/user_manual/introduction_qgis_gui_start_up.qgs]: You are right!. That was not intentional.

Thanks!!

**Comment by SrNetoChan** [comment]
@DelazJ @yjacolin this is ready for review. Thanks

File: qgis-projects/user_manual/macro_example.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
+def openProject():
+    pass
+
+def saveProject():
+	msgbar = iface.messageBar()
+	msgbar.PushMessage("WARNING","Project contains sensitive data."
+	                   " Do not publish.",
+					   QgsMessageBar.WARNING, 10)
```
Comment (SrNetoChan) [inline qgis-projects/user_manual/macro_example.py]: I have just used the same example that has been there for some time. I think it's only an example, it does not affect how users will perceive the functionality. IMHO

File: qgis-projects/user_manual/working_with_rasters.qgs [hunk 3]
```diff
@@ -81,55 +78,15 @@
...
-    <maplayer minimumScale="-4.65661e-10" maximumScale="1e+08" type="raster" hasScaleBasedVisibilityFlag="0">
+      <extent>
+        <xmin>-6232946.67269766889512539</xmin>
+        <ymin>-735684.66176720894873142</ymin>
+        <xmax>6363148.43763725832104683</xmax>
+        <ymax>9275122.96868141554296017</ymax>
+      </extent>
       <id>SR_50M_alaska_nad20160314120639764</id>
-      <datasource>../../../qgis_sample_data/raster/SR_50M_alaska_nad.tif</datasource>
+      <datasource>../../../../Geodata/qgis_sample_data/raster/SR_50M_alaska_nad.tif</datasource>
```
Comment (SrNetoChan) [inline qgis-projects/user_manual/working_with_rasters.qgs]: fixed

File: qgis-projects/user_manual/working_with_rasters.qgs [hunk 4]
```diff
@@ -1,29 +1,28 @@
...
-<qgis projectname="" version="2.15.0-Master">
+<qgis projectname="" version="2.18.9">
   <title></title>
-  <layer-tree-group expanded="1" checked="Qt::PartiallyChecked" name="">
+  <autotransaction active="0"/>
+  <evaluateDefaultValues active="0"/>
+  <layer-tree-group expanded="1" checked="Qt::Checked" name="">
     <customproperties/>
-    <layer-tree-layer expanded="1" checked="Qt::Unchecked" id="Colored_map20160314121544639" name="Colored_map">
+    <layer-tree-layer expanded="1" providerKey="gdal" checked="Qt::Checked" id="SR_50M_alaska_nad20160314120639764" source="/home/alexandre/qgis/qgis_sample_data/raster/SR_50M_alaska_nad.tif" name="SR_50M_alaska_nad">
```
Comment (SrNetoChan) [inline qgis-projects/user_manual/working_with_rasters.qgs]: It's relative... not sure what is the problem

File: qgis-projects/user_manual/introduction_qgis_gui_start_up.qgs [hunk 5]
```diff
@@ -30,31 +32,36 @@
...
-      <title></title>
-      <abstract></abstract>
+      <extent>
+        <xmin>-7117600</xmin>
+        <ymin>1367760</ymin>
+        <xmax>4897040</xmax>
+        <ymax>7809680</ymax>
+      </extent>
+      <id>landcover20170531225059200</id>
+      <datasource>../../../../Geodata/qgis_sample_data/raster/landcover.img</datasource>
```
Comment (SrNetoChan) [inline qgis-projects/user_manual/introduction_qgis_gui_start_up.qgs]: Fixed

**Comment by SrNetoChan** [comment]
@DelazJ ,

Thanks for the review.

> Why are there a diagrams.qgs and a vector_diagrams.qgs (nota: I didn't build the branch) ?

One was no longer needed. I have removed it.

> There's a readme text in the qgis_projects/user_manual folder in which I previously listed the existing qgis projects. Maybe could be worth keeping it uptodate with these changes. Could also act as minimal metadata for these files (to easily identify which one to use, regarding the screenshot you want to create/update).

I was me who created this folder and readme file. But I failed to keep it update. I have rename it as markdownfile so that can be read in github and I tried  to update it as best as I could without going file by file checking what it was used for.

> Can you also update the editnewvectorlayer (actually, it's shapefile), the addpostgistables and newpostgisconnection (Windows platform) image files?

Could not find editnewvectorlayer, but updated the other two.

**Comment by DelazJ** [comment]
> Could not find editnewvectorlayer

Sorry for the mistake, it's actually editNewVector in managing_data_source folder.

**Comment by SrNetoChan** [comment]
@DelazJ thanks for pointing me to those figures, I didn't notice the change (I try to open every dialog and compare but...)

Let me know if it's ok to merge this.

**Comment by DelazJ** [comment]
> @DelazJ thanks for pointing me to those figures, I didn't notice the change

I added the PG dialogs recently, while refactoring the Structure data chapter, reason why I'm aware. and noticed at that moment the fixed dialog title for shapefiles.
In the new shapefile screenshot, the `Precision` column is hidden. Is that on purpose? Otherwise, it'd be nice to show it with a value (which wasn't the case in the previous screenshot)

> Let me know if it's ok to merge this.

Other than my above comment, I'm ok with merging the PR (though I suspect some remaining custom links in qgs files but it's really not a big issue - anyone knows how to handle bad paths and it will be fixed if ever the issue arises. Also, you are the one who uses the most those files so it's okay)

**Comment by DelazJ** [comment]
Thanks @SrNetoChan

**Pull request merged**